bin/deepspeed.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+python "%~dp0\ds" %*
```

Comment:
Looks like the formatter wants a newline at the end of these .bat files, that shouldn't cause any issues, right?

Reply:
No, that would work fine, let me fix it.